docs/README.md

```diff
@@ -65,43 +51,20 @@ The Shelf is a theme block responsible for showing a **collection of products**
-| `hideUnavailableItems`     | `Boolean`                        | Hides items that are unavailable.                                                                                                                                                                                                                      | `false`           |
-| `skusFilter`               | `SkusFilterEnum`                 | Control SKUs returned for each product in the query. The less SKUs needed to be returned, the more performant your shelf query will be.                                                                                                    | `"ALL_AVAILABLE"` |
-| `paginationDotsVisibility` | `Enum`                           | Controls if pagination dots below the Shelf should be rendered or not. Possible values: `visible` (always show), `hidden` (never show), `desktopOnly`, `mobileOnly`                                                                                    | `visible`         |
-| `navigationStep` | `number` &#124; `'page'` | How many elements should pass when you click in the arrows to navigate | `'page'` |
-| `productList`              | `ProductListSchema`              | Product list schema. For its configuration, you can check the `ProductListSchema` table below.                                                                                                                                                                                                           | -                 |
-| `trackingId` | `string` | Name to show in the Google Analytics | If nothing is passed it will use the name of the block instead |
-
-- For `SkusFilterEnum`:
-
-| Name | Value | Description |
-| ---- | ----- | ----------- |
-| First Available | `FIRST_AVAILABLE` | Most performant, ideal if you do not have a SKU selector in your shelf. Will return only the first available SKU for that product in your shelf query. |
-| All Available | `ALL_AVAILABLE` | A bit better performace, will only return SKUs that are available, ideal if you have a SKU selector but still want a better performance. |
-| All | `ALL` | Returns all SKUs related to that product, least performant option. |
-
-- For `SpecificationFilterItem`:
-
-| Prop name | Type     | Description                                      | Default value |
-| --------- | -------- | ------------------------------------------------ | ------------- |
-| `id`      | `String` | ID of Specification Filter to be searched for    | ""            |
-| `value`   | `String` | Value of Specification Filter to be searched for | ""            |
-
-
-It is possible to add a related products shelf in a Product Details Page. Once its content depends on product data, the `RelatedProducts`block can only be declared in a product template (`store.product`). 
-
-### `RelatedProducts`:
+### `RelatedProducts`
```

Comment:
No need for this title. The table can be after the code for the related products shelf